matches/dao/match-dao.go

```diff
@@ -0,0 +1,172 @@
+package dao
+
+import (
+	// pq acts as the driver for SQL requests
+	"database/sql"
+	"errors"
+
+	_ "github.com/lib/pq"
+)
+
+const connStr = "user=postgres dbname=postgres host=matches-db sslmode=disable"
+
+// MatchGetResponse contains the information stored about a given match
+type MatchGetResponse struct {
+	ID        int
+	userOne   int
+	userTwo   int
+	matchedOn string
+}
+
+// MatchListResponse contains the information stored about all matches
```

Comment:
Is there any point in opening every comment with the function/type name?

Reply:
Required by GoFmt

Reply:
Think this is required by the go formatter